runners/java-fn-execution/src/main/java/org/apache/beam/runners/fnexecution/control/DockerJobBundleFactory.java

```diff
@@ -86,6 +89,11 @@ public static DockerJobBundleFactory create(JobInfo jobInfo) throws Exception {
     IdGenerator stageIdGenerator = IdGenerators.incrementingLongs();
     ControlClientPool clientPool = MapControlClientPool.create();
 
+    // Register standard file systems.
+    // TODO Use actual pipeline options.
+    FileSystems.setDefaultPipelineOptions(
```

Comment:
This is a strange place to register filesystems. It's also strange this is necessary in order to make the default set of filesystems available. Is this the only way to do it?

Reply:
Yeah it should be during job server startup for sure. I thought we agreed on this in the previous similar PR?.. I may be misremembering.

Reply:
So, since this code is not running on the job server but instead is running on Flink workers, then I think the best place to put this is in FlinkExecutableStageFunction. DockerJobBundleFactory is still not a good place - this is not Docker-specific.